frontend/components/Button/Button.tsx

```diff
@@ -0,0 +1,57 @@
+import Link from "next/link";
+
+export type ButtonProps = {
+    /** The text inside the Button */
+    children: string
+    /** Which type of button we want, tertiary is a link */
```

Comment:
Why is tertiary always a link? What is the purpose of this API design?

Reply:
I would add the option to have an onclick event on it as well. Sometimes you want to run logic while moving somewhere. I would not limit the possibility of the API design here.

Reply:
Ah okay. I miss readed the code. So you can add onClick to tertiary option? If yes, resolve the conversation. My bad.